src/components/Modal/__tests__/Modal.spec.tsx

```diff
@@ -1,39 +1,46 @@
 import React from 'react';
-import toJson from 'enzyme-to-json';
+import { render, RenderResult, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
 import ReactModal from 'react-modal';
 import { Modal } from '../Modal';
 
 describe('Modal', () => {
+    let view: RenderResult;
+
     it('should render correctly', () => {
-        const wrapper = shallow(
-            <Modal isOpen contentLabel="Content label">
+        view = render(
+            <Modal isOpen contentLabel="Content label" ariaHideApp={false}>
                 Some children
             </Modal>
         );
-        expect(toJson(wrapper)).toMatchSnapshot();
+
+        const dialog = screen.getByRole('dialog');
+
+        expect(view.baseElement).toMatchSnapshot();
+        expect(dialog).toBeInTheDocument();
     });
+
     it('should apply classnames from props correctly', () => {
-        const wrapper = shallow(
+        view = render(
             <Modal
                 isOpen
                 contentLabel="Content label"
                 className="content-class"
                 overlayClassName="overlay-class"
                 portalClassName="portal-class"
+                ariaHideApp={false}
```

Comment:
Added ariaHideApp to fix 
![image](https://github.com/textkernel/oneui/assets/104063210/6b20f72b-b32f-4714-a832-841bacb9bc04)